src/azure-cli/azure/cli/command_modules/cognitiveservices/tests/latest/test_account_properties.py

```diff
@@ -55,6 +55,13 @@ def test_cognitiveservices_account_public_network_access(self, resource_group):
         print(account)
         self.assertEqual(account['properties']['publicNetworkAccess'], 'Enabled')
 
+        import time
+        for i in range(10):
+            time.sleep(15)
```

Comment:
Why 15?

Reply:
In another similar cognitiveservices test, a comment says 
> when generating recording, use a large value such as 15

We can put the wait and retry logic in testsdk later so we don't need to write the loop ourselves.